routes/web.php

```diff
@@ -9,49 +9,32 @@
  * @see \Rogue\Providers\RouteServiceProvider
  */
 
-// Homepage
-$router->get('/', 'Legacy\Web\PagesController@home')->name('login');
+// Homepage & FAQ
+$router->view('/', 'pages.home')->middleware('guest')->name('login');
+$router->view('faq', 'pages.faq');
 
 // Authentication
 $router->get('login', 'Auth\AuthController@getLogin');
 $router->get('logout', 'Auth\AuthController@getLogout');
 
 // Campaigns
-$router->get('campaigns', 'Legacy\Web\AshesCampaignsController@index');
-$router->get('campaigns/{id}/inbox', 'Legacy\Web\AshesCampaignsController@showInbox');
-$router->get('campaigns/{id}', 'Legacy\Web\AshesCampaignsController@showCampaign')->name('campaigns.show');
-// Create, update, delete campaigns via Rogue.
-// @TODO: rename these routes (to just /campaigns) when we get off Ashes.
-$router->resource('campaign-ids', 'Legacy\Web\CampaignsController');
-
-// Exports
-$router->get('exports/{id}', 'Web\ExportController@show');
+$router->get('campaigns', 'Legacy\Web\CampaignsController@index');
+$router->get('campaigns/{id}', 'Legacy\Web\CampaignsController@show')->name('campaigns.show');
 
-// Posts
-$router->post('posts', 'Legacy\Web\PostController@store');
-$router->get('posts', 'Legacy\Web\PostController@index');
-$router->delete('posts/{id}', 'Legacy\Web\PostController@destroy');
+// Create, update, delete campaigns via Rogue.
+// @TODO: Merge into CampaignsController, above.
+$router->resource('campaign-ids', 'Legacy\Web\CampaignIdsController');
```

Comment:
I'm confused about how this bit works, can you walk me through it?

Reply:
oh wait I totally missed this it is still here! This makes sense though to me because it has to be different then showing the other campaign pages (overview, inbox) - sorry if I caused confusion!

Reply:
Yeah, totally!

Right now, we have a `/campaigns` resource which has an "index" of campaigns & their activity, and a "show" resource with posts for a particular campaign. Then we have a separate `/campaign-ids` resource which also has campaign "index" and "show" pages, but only for information about the campaign itself.

Instead of having separate `campaign` and `campaign-id` resources, I think we could merge these overlapping views together and both simplify the taxonomy (no longer two things that refer to campaigns) and also make the admin flow a little more straightforward.

Reply:
I'm not sure when we'd get to that work, though, so maybe not the most applicable `TODO`! 🙃

Reply:
AHHH I got it so this controller is only for the `campaign`s themselves? The other controller is for their activity? Could probably do some renaming down the line to make that a little less confusing but totally don't need to tackle now

Reply:
Yeah, exactly! And yeah, it's definitely confusing right now!